### ISSUE TITLE
Code reference should have the handbook sidebar.

### ISSUE THREAD

**Issue by tellyworth** [WordPress/wporg-developer#29]
### Discussed in https://github.com/WordPress/wporg-developer/discussions/1

<div type='discussions-op-text'>

<sup>Originally posted by **StevenDufresne** May 23, 2022</sup>
Related to https://github.com/WordPress/wporg-developer/discussions/2.

This makes the argument that the Code Reference should have the same sidebar as the [handbooks](https://developer.wordpress.org/block-editor/).

## Why
- The code already exists
- We want to create a consistent wp.org experience
- We can improve the handbooks at the same time.

Here's how it could look like:

<img src="https://d.pr/i/MyouCb.png" width="500">

### Potential Improvements:
There are tickets in the meta repo that speak to this, although I don't have them in hand.

1. Use more of the page.
2. Fix the way items are visually nested
3. Smarter logic to expand the active section (to expand you have to click the arrow down, which is small,... clicking the title does something different)
4. Related: Make the search results appear in-line
</div>

**Comment by apeatling** [comment]
It would be a huge improvement to remove the third level nesting in the navigation, it's way too much to have that nested in the sidebar. Lots of examples on the Gutenberg docs.

Instead use the top second level items as an overview pages that list out the third level items. Then make sure there's a back or breadcrumb link back on the third level item pages.

**Comment by adamwoodnz** [comment]
This didn't make it into the current redesign. @StevenDufresne is this still something we should raise with design, or should it be closed?

**Comment by StevenDufresne** [comment]
I don't think it has priority right now, but I'm personally in favor of _something_ that makes navigating functions more straightforward.
